data/gui/widget/spinner_default.cfg

```diff
@@ -0,0 +1,220 @@
+#textdomain wesnoth-lib
+###
+### Definition of a spinner widget.
+###
+
+#define GRID_SPLIT ALIGN
+	[grid]
+		id = "_content_grid"
+
+		[row]
+			grow_factor = 0
+			[column]
+				horizontal_alignment={ALIGN}
+				[grid]
+					[row]
+						[column]
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_prev"
+								definition = "left_arrow"
+							[/button]
+						[/column]
+
+						[column]
+							border = "left,right"
+							border_size = 3
+							horizontal_alignment={ALIGN}
+							[text_box]
+								id = "_text"
+							[/text_box]
+						[/column]
+
+						[column]
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_next"
+								definition = "right_arrow"
+							[/button]
+						[/column]
+					[/row]
+				[/grid]
+			[/column]
+		[/row]
+	[/grid]
+#enddef
+
+#define GRID_LEFT ALIGN
+	[grid]
+		id = "_content_grid"
+
+		[row]
+			grow_factor = 0
+			[column]
+				horizontal_alignment={ALIGN}
+				[grid]
+					[row]
+						[column]
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_prev"
+								definition = "left_arrow"
+							[/button]
+						[/column]
+
+						[column]
+							border = "left"
+							border_size = 3
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_next"
+								definition = "right_arrow"
+							[/button]
+						[/column]
+
+						[column]
+							border = "left"
+							border_size = 3
+							horizontal_alignment={ALIGN}
+							[text_box]
+								id = "_text"
+							[/text_box]
+						[/column]
+
+					[/row]
+				[/grid]
+			[/column]
+		[/row]
+	[/grid]
+#enddef
+
+#define GRID_RIGHT ALIGN
+	[grid]
+		id = "_content_grid"
+
+		[row]
+			grow_factor = 0
+			[column]
+				horizontal_alignment={ALIGN}
+				[grid]
+					[row]
+
+						[column]
+							border = "right"
+							border_size = 3
+							horizontal_alignment={ALIGN}
+							[text_box]
+								id = "_text"
+							[/text_box]
+						[/column]
+						
+						[column]
+							border = "right"
+							border_size = 3
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_prev"
+								definition = "left_arrow"
+							[/button]
+						[/column]
+
+						[column]
+							horizontal_alignment={ALIGN}
+							[button]
+								id= "_next"
+								definition = "right_arrow"
+							[/button]
```

Comment:
I think these should probably be `[repeating_button]`.

Reply:
Yes, you are right, that seems to be the better solution.